lib/pages/chat_page.dart

```diff
@@ -175,55 +174,55 @@ class _ChatPageState extends State<ChatPage> {
-    AuthGoogle authGoogle =
-        await AuthGoogle(fileJson: "assets/abdo-bot-briv-9ed7278f51f5.json")
-            .build();
-    Dialogflow dialogflow = Dialogflow(authGoogle: authGoogle);
-    AIResponse response = await dialogflow.detectIntent(query);
-    List<String> chatTraining = _chat.getChatTraining();
-    for (var i = 0; i < chatTraining.length; i++) {
-      if (response.getMessage() == chatTraining[i]) {
-        _chat.result = true;
-        // result = true;
-        setState(() {
-          _messageModel.insertMessage(
-              0, ChatTraining(answer: response.getMessage()));
-        });
-        break;
-      } else {
-        _chat.result = false;
-        // result = false;
-      }
-    }
-    if (response.getMessage() != null &&
-        response.getMessage() != "" &&
-        _chat.result == false) {
-      Message message = Message(
-        text: response.getMessage(),
-        name: "Bot",
-        type: false,
-      );
-      setState(() {
-        _messageModel.insertMessage(0, message);
-      });
-    }
+    // _textController.clear();
+    // AuthGoogle authGoogle =
+    //     await AuthGoogle(fileJson: "assets/abdo-bot-briv-9ed7278f51f5.json")
+    //         .build();
+    // Dialogflow dialogflow = Dialogflow(authGoogle: authGoogle);
+    // AIResponse response = await dialogflow.detectIntent(query);
+    // List<String> chatTraining = _chat.getChatTraining();
+    // for (var i = 0; i < chatTraining.length; i++) {
+    //   if (response.getMessage() == chatTraining[i]) {
+    //     _chat.result = true;
+    //     // result = true;
+    //     setState(() {
+    //       _messageModel.insertMessage(
+    //           0, ChatTraining(answer: response.getMessage()));
+    //     });
+    //     break;
+    //   } else {
+    //     _chat.result = false;
+    //     // result = false;
+    //   }
+    // }
+    // if (response.getMessage() != null &&
+    //     response.getMessage() != "" &&
+    //     _chat.result == false) {
+    //   Message message = Message(
+    //     text: response.getMessage(),
+    //     name: "Bot",
+    //     type: false,
+    //   );
+    //   setState(() {
+    //     _messageModel.insertMessage(0, message);
+    //   });
+    // }
   }
 
   void _handleSubmitted(String text) {
-    if (text.isEmpty) {
-      print("empty message");
-    } else {
-      Message message = Message(
-        text: text,
-        name: "Patient",
-        type: true,
-      );
-      setState(() {
-        _messageModel.insertMessage(0, message);
-      });
-      response(text);
-      _textController.clear();
-    }
+    // if (text.isEmpty) {
+    //   print("empty message");
+    // } else {
+    //   Message message = Message(
+    //     text: text,
+    //     name: "Patient",
+    //     type: true,
+    //   );
+    //   setState(() {
+    //     _messageModel.insertMessage(0, message);
+    //   });
+    //   response(text);
+    //   _textController.clear();
+    // }
```

Comment:
These lines will be fixed by the next pr right ?

Reply:
Yes